examples/seismic/model.py

```diff
@@ -98,6 +150,89 @@ def demo_model(preset, **kwargs):
         return Model(vp=v, origin=origin, shape=v.shape,
                      spacing=spacing, nbpml=20)
 
+    elif preset.lower() in ['marmousitti', 'marmousi2dtti']:
+
+        dimensions = (201, 201, 70)
```

Comment:
why do we need both ``dimensions`` and ``shape``? btw, usually (not always, unfortunately, but most of the times yes, I think) we use ``dimensions`` to represent the symbolic dimensions ``x, y, z``; here it's just numbers

Reply:
A legacy convention from the early seismic prototypes, before `Dimension` objects were introduced. Should probably be changed everywhere (loads more outside TTI), but probably beyond this PR; and should disappear once a `Grid` class gets introduced.